packages/odyssey-react/src/components/Infobox/Infobox.test.tsx

```diff
@@ -28,6 +28,15 @@ describe("Infobox", () => {
     expect(screen.getByText(actions)).toBeVisible();
   });
 
+  it("restricts title and content props via types", () => {
+    // @ts-expect-error requires title or content prop
+    <Infobox />;
+
+    <Infobox title={title} />;
+    <Infobox content={content} />;
+    <Infobox title={title} content={content} />;
+  });
```

Comment:
Im not sure im ever going to get used to this 😅

Reply:
Which part? happy to talk further